contents/blog/ceo-diary-2.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2021-09-10
+title: Optimize for not breaking up with your co-founder
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ceo-diaries
+author: james-hawkins
+featuredImage: ../images/blog/ceo-diary.png
+featuredImageType: full
+---
+
+Last week we had an offsite in the beautiful Portuguese countryside, an hour's bus journey from Porto. You should [watch the TikTok](https://vm.tiktok.com/ZMRUGX4XW/).
+
+One of the sessions we ran was a PostHog pre-mortem. This involved breaking up into teams and getting people to suggest what is most likely to kill the company. What a cheerful way to start the week...
+
+What surprised me was the number of people that listed cofounder breakup as a potential cause of failure.
+
+And they're right.
+
+Around [20% of the YC funded startups have a founder leave](http://paulgraham.com/startupmistakes.html).
+
+So, I took an action, as any good PostHog offsite attendee does, to cover what Tim and I are doing to not break up.
+
+# We have reduced opportunities for things to get into a bad spiral
+
+We talk frequently and at length:
+- We talk 1/1 for at least an hour a week, although often, far more.
+- We record any stuff we want to cover during the week in a shared agenda.
+- We talk about frustrations / fears, as well as the positives.
+
+We pay ourselves the same. Else it just creates a chance for resentment. Especially around equity.
+- We have the same equity as each other.
+- We are paid the same as each other.
+
+We are clear who is doing what.
+- There are a handful of things we _both_ want to be involved in. People is one, Strategy is another. We think both of us being involved leads to the best decisions here.
+- Other things we have clear boundaries. Tim codes and works with engineering, I sell and work with design and our marketing team on positioning.
+
+# Check you value the same things
```

Comment:
Should this be 'We value the same things' to keep it consistent with the heading before.

Reply:
Changed the structure a bit here - made the first list bullet points (there are three points within the first subheader now), so this header went away